src/plugins/ui_stateful/edition.ts

```diff
@@ -755,14 +755,18 @@ export class EditionPlugin extends UIPlugin {
 
     let values: string[];
     if (rule.criterion.type === "isValueInList") {
-      values = rule.criterion.values;
+      values = [...new Set(rule.criterion.values)];
```

Comment:
nitpick: filter duplicates values after the if/else rather than doing it in each case